timur/lib/client/jsx/components/document/document_view.jsx

```diff
@@ -19,7 +19,7 @@ export default class DocumentView extends React.Component {
   }
 
   render(){
-    let { document, onUpdate, editing, children } = this.props;
+    let { document, documentName, onUpdate, editing, children } = this.props;
```

Comment:
nit: overwriting the global `document` here and in the related components seems slightly risky? Should this be named something else?

Reply:
This is my bad from way back. It probably doesn't break anything except the local scope, we can refactor it out later (these components may not survive vulcan).

Reply:
It's an argument for a low scope linter, too, I think.  I'm not a fan of many linters, but it's possible to have a linter that can also _auto correct_ and focus on just the big issues (like shadowing an important global) and get some big value out of it.